rpmlint/lint.py

```diff
@@ -224,6 +224,9 @@ def validate_files(self, files):
         # check all elements if they are a folder or a file with proper suffix
         # and expand everything
         packages = self._expand_filelist(files)
+
+        # Sort the files so that the output is stable
+        packages = sorted(packages)
```

Comment:
We probably want this forced to C locale for sorting right?

Reply:
Well, we need any stable sort and we have other places where we use just `sorted` without any specific locale.